.travis.yml

```diff
@@ -42,3 +42,4 @@ branches:
   only:
     - master
     - /^greenkeeper/.*$/
+    - labs
```

Comment:
Should it be `/^labs/.*$/`?

Reply:
True, much simpler than manually add it every time.